Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -1000,10 +1000,15 @@ ShaderCode GeneratePixelShaderCode(APIType api_type, const ShaderHostConfig& hos
   else
 #endif
   {
-    out.Write("{} {} {} {};\n", "FRAGMENT_OUTPUT_LOCATION_INDEXED(0, 0)",
-              use_framebuffer_fetch ? "FRAGMENT_INOUT" : "out",
-              uid_data->uint_output ? "uvec4" : "vec4",
-              use_framebuffer_fetch ? "real_ocol0" : "ocol0");
+    if (use_framebuffer_fetch)
+    {
+      out.Write("FRAGMENT_OUTPUT_LOCATION(0) FRAGMENT_INOUT vec4 real_ocol0;\n");
```

Comment:
Is framebuffer fetch always a float?  We used to support it being an int as well.

Reply:
uint_output is used for logic ops, but if framebuffer fetch is in use, it should be replacing hardware logic ops
(The (unofficial) Intel Metal fbfetch actually breaks if you try to combine it with any blending, so we definitely have to make sure this is the case)
(Also uint_output is only used on dx which doesn't have framebuffer fetch, but my main reason is the above)